### DISCUSSION TITLE
Errors in dev vs production ?

### DISCUSSION THREAD

**Discussion by MihaiWill** [vercel/next.js#15059] [Help] [answered]
I got this wanted error on graphql when login is not valid:
![alt text](https://imgur.com/HE6KDZy.png "error")

Under login modal, there is the error modal in development. I assume in production, there will be only the console error, and everything will be fine? 

Just to confirm, this is right behavior and I can continue like this, right?

**Comment by jamesmosier** [comment, marked as answer]
Correct, the error overlay is development only.

---

I'd recommend catching the error thought via `try{}catch(e){}` or some other similar method.

**Reply by MihaiWill** [reply]
Thanks for recommendation! I will
